fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/LdpContainerRdfContext.java

```diff
@@ -203,9 +203,13 @@ public FedoraResourceTripleFunction(final String insertedContainerProperty,
                 return Iterators.transform(values, new Function<Value, Triple>() {
                     @Override
                     public Triple apply(final Value input) {
-                        final RDFNode membershipResource = new ValueConverter(session(), translator())
-                                .convert(input);
-                        return create(subject(), memberRelation, membershipResource.asNode());
+                        try {
+                            final RDFNode membershipResource = new ValueConverter(resource().getNode().getSession(),
```

Comment:
I dimly, dimly remember this being the intent for that field; that it would be a convenient handle for the `Session` in context without constantly dealing with JCR's annoying `RepositoryException` addiction.